llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -55,15 +56,91 @@ void InterBlockScheduling::markEpilogueBlocks() {
   }
 }
 
+std::map<MachineBasicBlock *, LivePhysRegs> calculateLiveOuts(MachineFunction *MF) {
+  std::queue<MachineBasicBlock *> WorkList;
```

Comment:
As there is no change withing basic blocks and instructions, you can use `const` everywhere.